packages/hotjar/package.json

```diff
@@ -38,7 +38,8 @@
   "sideEffects": false,
   "devDependencies": {
     "@edx/browserslist-config": "1.1.0",
-    "@edx/frontend-build": "12.3.0",
+    "@edx/frontend-build": "12.7.0",
+    "@edx/frontend-platform": "4.0.1",
```

Comment:
[clarification] Why is `@edx/frontend-platform` installed in `hotjar` when `hotjar` doesn't use it? Keep in mind this repo is a monorepo and each package within the monorepo has a unique set of dependencies.

Reply:
Yes, that was my bad as these were updated through a script